src/biz/bokhorst/xprivacy/XParam.java

```diff
@@ -29,6 +29,11 @@ public static XParam fromXposed(MethodHookParam param) {
 		xparam.args = param.args;
 		xparam.mResult = param.getResult();
 		xparam.mThrowable = param.getThrowable();
+		
+		if (xparam.args == null) {
```

Comment:
I got some errors in logcat about trying to count null array on xparam.args. It might be a small error in the Lollipop Xposed version where the Xposed args array is null. It did not harm I think, other than printing errors once awhile.